lib/her/model/base.rb

```diff
@@ -28,6 +28,17 @@ def [](attribute_name)
       def singularized_resource_name
         self.class.name.split('::').last.tableize.singularize
       end
+
+      # @private
+      def initialize_dup(other)
+        @_her_attributes = @_her_attributes.dup
+        @_her_attributes[self.class.primary_key] = nil
+
+        @new_record  = true
```

Comment:
we can remove this since [new_record?](https://github.com/remiprev/her/blob/master/lib/her/model/orm.rb#L13) doesn't rely on an instance variable 😺